glog.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: glog
+  version: 0.6.0
+  epoch: 0
+  description: "C++ implementation of the Google logging module"
+  copyright:
+    - license: "BSD-3-Clause"
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - cmake
+      - build-base
+      - libucontext-dev
+      - libunwind-dev
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/google/glog
+      tag: v${{package.version}}
+      expected-commit: b33e3bad4c46c8a6345525fd822af355e5ef9446
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_INSTALL_LIBDIR=lib \
+        -DBUILD_SHARED_LIBS=ON \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DWITH_GFLAGS=ON \
+        -DWITH_THREADS=ON \
+        -DWITH_TLS=ON \
+        -DPRINT_UNSYMBOLIZED_STACK_TRACES=OFF \
+        -G Ninja
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: "glog-dev"
+    description: "C++ implementation of the Google logging module (development files)"
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - glog
+
+update:
+  enabled: true
+  github:
+    identifier: google/glog
+    use-tag: true
```

Comment:
It seems this project uses releases -- should `use-tag` be omitted so the releases are pulled instead?

Reply:
ahh, i see, i will drop that

Reply:
done